src/sftp/handler.go

```diff
@@ -45,31 +45,30 @@ func NewSftpHandler(credentialGetter secrets.CredentialGetter) (*SftpHandler, er
 		return nil, err
 	}
 
-	sftpUserNameSecret := "sftp-user-" + utils.EnvironmentName() // pragma: allowlist secret
+	sftpUserNameSecret := "ca-phl-sftp-user-" + utils.EnvironmentName() // pragma: allowlist secret
 	sftpUser, err := credentialGetter.GetSecret(sftpUserNameSecret)
 	if err != nil {
 		slog.Error("Unable to get SFTP username secret", slog.String("KeyName", sftpUserNameSecret), slog.Any(utils.ErrorKey, err))
 		return nil, err
 	}
 
-	sftpPasswordSecret := "sftp-password-" + utils.EnvironmentName() // pragma: allowlist secret
+	sftpPasswordSecret := "ca-phl-sftp-password-" + utils.EnvironmentName() // pragma: allowlist secret
 	sftpPassword, err := credentialGetter.GetSecret(sftpPasswordSecret)
 	if err != nil {
 		slog.Error("Unable to get SFTP password secret", slog.String("KeyName", sftpPasswordSecret), slog.Any(utils.ErrorKey, err))
 		return nil, err
 	}
 
-	// TODO uncomment code when partner is setup to receive key
 	config := &ssh.ClientConfig{
 		User: sftpUser,
 		Auth: []ssh.AuthMethod{
-			//ssh.PublicKeys(pem),
+			ssh.PublicKeys(pem),
 			ssh.Password(sftpPassword),
```

Comment:
From what CADPH told us on Friday, I think we'll actually want to take out the password stuff and just do username and keys

Reply:
Removed password!